litellm/llms/fireworks_ai/common_utils.py

```diff
@@ -20,6 +20,15 @@ class FireworksAIMixin:
     def get_error_class(
         self, error_message: str, status_code: int, headers: Union[dict, Headers]
     ) -> BaseLLMException:
+        # Check if it's a rate limit error (status code 429)
+        if status_code == 429:
```

Comment:
this logic / handling should be placed here: https://github.com/BerriAI/litellm/blob/7262606411bc974102fb366258e0681b978e3777/litellm/litellm_core_utils/exception_mapping_utils.py#L246

Reply:
done! - move it to the right file now